targets/TARGET_NUVOTON/TARGET_M2354/TARGET_TFM/TARGET_NU_M2354/COMPONENT_TFM_S_FW/CMakeLists.txt

```diff
@@ -8,20 +8,35 @@ target_link_libraries(mbed-m2354-tfm
         ${CMAKE_CURRENT_SOURCE_DIR}/s_veneers.o
 )
 
-mbed_post_build_nuvoton_tfm_sign_image_tgt(
-    NU_M2354
-    ${CMAKE_CURRENT_SOURCE_DIR}
-    nuvoton_m2354-root-rsa-3072.pem
-)
-
+# Must align with TF-M import assets
 set(bl2 true)
-set(mcuboot_image_number 1)
+set(mcuboot_image_number 2)
 set(region_defs_h_path "${CMAKE_CURRENT_SOURCE_DIR}/partition/region_defs.h")
+set(update_stage_sdh true)
+set(update_stage_flash false)
```

Comment:
Does it mean we support both single and multi image boots, and updating from SD card and flash? Have all of them been tested with Mbed OS? And how will users select which one to use?

Reply:
Both from SD and from flash are supported and they are both verified with Mbed OS. But due to limited flash size 1MiB on M2354, from flash is not practical. From SD is the default and for public. From flash is for internal development/test.

Reply:
In principle, we can keep only the default/supported cased here, rather than having code for all cases. But I get the idea of making it flexible for internal development, so I'm personally fine with this.